src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMStep

```diff
@@ -0,0 +1 @@
+BitbucketSCM
```

Comment:
Ah, this looks weird in the diff, because it is a symlink. I was confused for a while at what I was looking at :grin: